frontend/src/components/pages/CourseOverview.tsx

```diff
@@ -1,25 +1,76 @@
 import React from "react";
-import { Box, Flex, VStack } from "@chakra-ui/react";
+import { useHistory, useParams } from "react-router-dom";
+import { ChevronLeftIcon } from "@chakra-ui/icons";
+import { Box, Button, Flex, Text, VStack } from "@chakra-ui/react";
+import { useQuery } from "@apollo/client";
 import Banner from "../learner/Banner";
 import ModuleLessonCount from "../learner/ModuleLessonCount";
+import { GET_COURSE } from "../../APIClients/queries/CourseQueries";
+import { COURSE_OVERVIEW_BASE_ROUTE } from "../../constants/Routes";
+import { CourseOverviewParams } from "../../types/CourseOverviewTypes";
+import { ModuleType } from "../../types/ModuleEditorTypes";
+
+const CourseOverview = (): React.ReactElement => {
+  const history = useHistory();
+  const { courseID }: CourseOverviewParams = useParams();
+  const { data: courseData } = useQuery(GET_COURSE, {
+    variables: {
+      id: courseID,
+    },
+  });
+
+  const onReturnToCourseOverviewClick = () => {
+    history.push(COURSE_OVERVIEW_BASE_ROUTE);
+  };
+
+  const getLessonCount = () => {
+    let lessonCount = 0;
+
+    courseData?.course?.modules?.forEach((moduleContent: ModuleType) => {
+      lessonCount += moduleContent?.lessons?.length;
+    });
+
+    return lessonCount;
+  };
 
-const ViewCourse = (): React.ReactElement => {
   return (
     <Flex direction="column">
       <Banner />
       <Flex
         px="120px"
         py="20px"
         height="400px"
-        background="background.dark"
+        background="background.lightgrey"
         color="white"
         align="center"
       >
         <VStack flex="2" align="start">
```

Comment:
Could you add a right-hand padding to this component as well? If the description overflows to be longer than a single line, the text is currently touching the card on the right. Example:
![image](https://user-images.githubusercontent.com/9922514/183266644-f4284e62-0d6d-4403-9f68-a334e5c01b73.png)